### PR TITLE
[RemoteConfig] Fix the default cache value

### PR DESCRIPTION
**Pull request by cynthiajoan**
The default cache should be in milliseconds

### CODE REVIEW

File: remote_config/src/include/firebase/remote_config.h [hunk 1]
```diff
@@ -115,8 +115,8 @@ enum ConfigSetting {
 };
 
 /// @brief The default cache expiration used by Fetch(), equal to 12 hours,
-/// in seconds.
-static const uint64_t kDefaultCacheExpiration = 60 * 60 * 12;
+/// in milliseconds.
+static const uint64_t kDefaultCacheExpiration = 60 * 60 * 12 * 1000;
```
Comment (vimanyu) [inline remote_config/src/include/firebase/remote_config.h:119]: nit: For clarity and consistency, should we name this "kDefaultCacheExpirationInMilliseconds" just like "kDefaultTimeoutInMilliseconds" ?

**Review by vimanyu — COMMENTED** [review]
minor nit but looks good otherwise.

File: remote_config/src/include/firebase/remote_config.h (hunk 1 above)
Comment (cynthiajoan) [reply remote_config/src/include/firebase/remote_config.h:119]: We could before, but I'm hesitating to change an api name now.

Comment (vimanyu) [reply remote_config/src/include/firebase/remote_config.h:119]: Good point. I guess we will leave it as it is for now then.

**Review by vimanyu — APPROVED** [review]
LGTM

**Pull request merged**